libyul/backends/wasm/polyfill/Memory.yul

```diff
@@ -66,4 +66,39 @@ function memset(ptr:i32, value:i32, length:i32) {
 	{
 		i32.store8(i32.add(ptr, i), value)
 	}
-}
\ No newline at end of file
+}
+
+function mload_internal(pos:i32) -> z1, z2, z3, z4 {
+	z1 := bswap64(i64.load(pos))
+	z2 := bswap64(i64.load(i32.add(pos, 8:i32)))
+	z3 := bswap64(i64.load(i32.add(pos, 16:i32)))
+	z4 := bswap64(i64.load(i32.add(pos, 24:i32)))
+}
+
+function mstore_address(pos:i32, a1, a2, a3, a4) {
```

Comment:
I think this is inconsistent, in `mload_address` you only deal with 3 return variables. I think it is fine to always use 4, mimicing EVM.